### PR TITLE
mgr/telemetry: add report_timestamp to sent reports

### PR DESCRIPTION
**Pull request by dmick**
Received time may differ from report time, and correlating
to local cluster state events might be useful.

Signed-off-by: Dan Mick <dan.mick@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/pybind/mgr/telemetry/module.py [hunk 1]
```diff
@@ -186,7 +186,8 @@ def gather_crashinfo(self):
     def compile_report(self):
         report = {
             'leaderboard': False,
-            'report_version': 1
+            'report_version': 1,
+            'report_timestamp': datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S.%f'),
```
Comment (sebastian-philipp) [inline src/pybind/mgr/telemetry/module.py]: `datetime.utcnow().isoformat() + 'Z'` ?

**Comment by liewegas** [comment]
can you rebase and retarget this at master?  then we can backport from there

File: src/pybind/mgr/telemetry/module.py (hunk 1 above)
Comment (dmick) [reply src/pybind/mgr/telemetry/module.py]: Are you suggesting this as a different way to format the time?  I was choosing the same format used by Ceph when sending crashdumps

Comment (dmick) [reply src/pybind/mgr/telemetry/module.py]: isoformat() does look like the same thing, but I then don't understand adding timezone

**Comment by dmick** [comment]
sure, will do

File: src/pybind/mgr/telemetry/module.py (hunk 1 above)
Comment (dmick) [reply src/pybind/mgr/telemetry/module.py]: changed to isoformat(), left off the Z

**Comment by dmick** [comment]
(this is now not a nautilus PR, so maybe the title should go back?..)

**Pull request merged**